docs/using-mirrord/mirrord-for-ci.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "mirrord for CI"
+description: "How to use mirrord in a CI environment with `mirrord ci` commands."
+date: 2025-10-21T00:00:00+03:00
+lastmod: 2025-10-21T00:00:00+03:00
+draft: false
+menu:
+  docs:
+    parent: "using-mirrord"
+toc: true
+tags:
+  - open source
+  - team
+  - enterprise
+---
+
+mirrord can be used to greatly speed up CI runs by avoiding the need to deploy the app you're trying to test.
+Instead of setting up a whole kubernetes environment for every CI run, you can just use mirrord to redirect
+the traffic from the service you want to test, which is already running in some cluster (e.g. a staging environment).
+
+While running regular `mirrord exec` can be made to work for this purpose, it requires some
+finagling to get right, such as wrapping `mirrord exec` in some other command that would start
+it as a background process (or some other approach to keep it running in the background).
+The `mirrord ci start` command is more appropriate for this use case, since it starts your app and mirrord as
+background processes, allowing you to run tests and whatever else you want while mirrord is running.
+
+Note that you can only have **one** mirrord CI session per service during a CI run, this means that
+we (currently) don't support executing multiple `mirrord ci start` commands. You must do one
+`mirrord ci start`, run your tests, then run `mirrord ci stop`, before you're able to run
+`mirrord ci start` again in the same CI run.
+
+The CI runner must be able to access the kubernetes cluster where the service you want to target is deployed,
+otherwise mirrord won't work. It's recommended that you create a kubernetes
+[service account](https://kubernetes.io/docs/concepts/security/service-accounts/) for the CI runner.
+
+## Prerequisites
+
+1. Minimum mirrord CLI version `3.166.0`.
+2. The CI runner must be able to access the kubernetes cluster (kubernetes api access to
+   the staging cluster, for example).
+
+## For mirrord Operator users
+
+To use mirrord ci with mirrord Operator, you'll need to generate a CI API key and store it
+as a **secret** environment variable.
```

Comment:
```suggestion
as a **secret** environment variable. This will prevent your usage of mirrord in CI from expending your seats, which are counted using a machine-based ID.
```